app/assets/javascripts/annotations.js

```diff
@@ -163,6 +164,72 @@ var initializeAnnotationsForCode = function() {
     return box;
   }
 
+  function newAnnotationBoxForPDF(annObj) {
+
+    var problemStr = annObj.problem_id? getProblemNameWithId(annObj.problem_id) : "General";
+    var valueStr = annObj.value? annObj.value.toString() : "None";
+    var commentStr = decodeURI(annObj.comment);
+
+    var grader = elt("span", {
+      class: "grader"
+    }, annObj.submitted_by + " says:");
+    var edit = elt("span", {
+      class: "edit glyphicon glyphicon-edit",
+      id: "edit-ann-" + annObj.id
+    });
+
+    var score = elt("div", {
+      class: "score-box"
+    }, elt("div", {}, "Problem: " + problemStr), elt("div", {}, "Score: " + valueStr));
+
+    var del = elt("span", {
+      class: "delete glyphicon glyphicon-remove"
+    });
+
+    if (isInstructor) {
+      var header = elt("div", {
+        class: "header"
+      }, grader, del, edit);
+    } else {
+      var header = elt("div", {
+        class: "header"
+      }, grader);
+    }
+
+    var body = elt("div", {
+      class: "body"
+    }, commentStr);
+
+    var box = elt("div", {
+      class: "ann-box",
+      id: "ann-box-" + annObj.id
+    }, header, body, score);
```

Comment:
Instead of reconstructing this box every time we need to accept user input, would it be easier to just have a `div` always in the page that is only visible when someone has clicked? It might make the JavaScript code look cleaner. We could populate the values as needed by manually setting the innerHTML.